src/main/handlers/media/transcribe.ts

```diff
@@ -0,0 +1,40 @@
+import { RuntimeProject } from 'sharedTypes';
+import voskTranscribeFunction from './transcriptionEngines/voskTranscribeFunction';
+import { TranscriptionEngine, TranscriptionFunction } from './transcribeTypes';
+import deepspeechTranscribeFunction from './transcriptionEngines/deepspeechTranscribeFunction';
+import dummyTranscribeFunction from './transcriptionEngines/dummyTranscribeFunction';
+
+const getTranscriptionFunction = new Map<
+  TranscriptionEngine,
+  TranscriptionFunction
+>();
+
+// Add the enum - function mapping for a transcription engine here
+getTranscriptionFunction.set(TranscriptionEngine.VOSK, voskTranscribeFunction);
```

Comment:
Can't you just replace this whole thing (lines 7-21) with:

```
const getTranscriptionFunction: Record<TranscriptionEngine, TranscriptionFunction> = {
  TranscriptionEngine.VOSK: voskTranscribeFunction,
   //... same for others
};
```

Any particular benefit to using `set`?

Reply:
Yep, that's a better solution. Fixed